src/components/Widgets/Sabnzbd.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+<div class="sabznbd">
+  <!-- Sabznbd Header -->
+  <div class="intro">
+    <p class="download">{{ download_speed }}</p>
+    <em :class="`fas fa-${status}`"></em>
+  </div>
+  <!-- Sabnzbd Details, If hideDetails set False -->
+  <div class="details" v-if="sabnzbdDetails.length > 0">
+    <div class="info-wrap" v-for="(section, indx) in sabnzbdDetails" :key="indx">
+      <p class="info-line" v-for="sabznbd in section" :key="sabznbd.label">
+          <span class="lbl">{{sabznbd.label}}</span>
+          <span class="val">{{ sabznbd.value }}</span>
+        </p>
+    </div>
+  </div>
+  <!-- Queue Details, If hideQueue set False -->
+  <div class="details" v-if="showQueue && sabnzbdQueue.length > 0">
+    <div class="info-wrap">
+      <p class="info-line" v-for="sabznbd in sabnzbdQueue" :key="sabznbd.label">
+        <em :class="`fas fa-${sabznbd.status}`"></em>
+        <span class="lbl">{{ sabznbd.filename }}</span>
+        <span class="lbl">{{ sabznbd.percentage }}%</span>
+      </p>
+    </div>
+  </div>
+  <!-- Show/ hide toggle button for Queue-->
+  <p class="more-details-btn" @click="toggleQueue" v-if="sabnzbdQueue.length > 0">
+    {{ showQueue ? "Hide Queue" : "Show Queue" }}
```

Comment:
In this instance, I think it's fine as is.
But you can also [make text translatable](https://github.com/Lissy93/dashy/blob/master/docs/multi-language-support.md#adding-new-text-to-a-component), by extracting it into [`en.json`](https://github.com/Lissy93/dashy/blob/master/src/assets/locales/en.json#L258), then doing:
`{{ showQueue ? $t('widgets.sabnzbd.hide-queue') : $t('widgets.sabnzbd.show-queue') }}`

And for the pause / play part on L69, that'd something like:
`return this.$t('widgets.sabnzbd.pause');`

Reply:
L29 makes sense but and will be updated in the next revision.

L69 (and L71) refers to the icon name listed for FontAwesome, not something that is displayed to the user. So I'm not 100% sure that this applies for those lines.